front/components/data_source_view/DataSourceViewSelector.tsx

```diff
@@ -199,7 +199,7 @@ export function DataSourceViewsSelector({
     item: DataSourceViewContentNode,
     prevState: DataSourceViewSelectionConfigurations
   ): DataSourceViewSelectionConfigurations {
-    const dsv = item.dataSourceView;
+    const dsv = item.dataSourceViews[0];
```

Comment:
noob question: don't we wanna make sure that we use the right dataSourceView here?

Reply:
As you make a search on a single space here, I assume we will always have one single datasourceView

Reply:
Fair, could we add a sanity check ? (for readability)

Reply:
In theory we could do a search on multiple spaces, there's no limitation on that - this would just open the first matching datasource, which i think is fine. Also dataSourceViews cannot be empty, there's always at least one value